### DISCUSSION TITLE
Getting nonsense text with HugingFace model gozfarb/pygmalion-7b-4bit-128g-cuda.

### DISCUSSION THREAD

**Discussion by redwoodhighway** [oobabooga/text-generation-webui#1795] [Q&A] [answered]
Have made several attempts to use oobabooga with various models but I seem to run into this issue every time. The latest model I have been trying is pygmalion-7b. I went through a guide and set the settings correctly but when I test all I get is strings of meaningless characters. I have reloaded the model and tested with other models and they seem to work. I also updated the cuda install on my system and updated oobabooga webui. Nothing seems to changed. The console windows just shows that it is replying no errors. I have a Nvidia GeForce GTX Titan X with 12GB of VRAM. Not sure if it's just to old to run those models. Any subjections would be appreciated.

Here is a short example

Me: Hello
Assistant: WebachivendorfŸign interioréné profilestamp kwietwall$)midnightmarrę Sierra Nevigue

**Comment by LaaZa** [comment, marked as answer]
I believe `.safetensors` format requires a newer GPTQ-for-LLaMA and causes this issue if you do not use a `.pt` format with your current version.

**Reply by redwoodhighway** [reply]
Thank you for this. I was able to look up GPTQ-for-LLaMA and found a redit post about it that had exact instructions on how to fix the issue.
Here is a link.
https://www.reddit.com/r/Oobabooga/comments/12suy4a/comment/jieewzy/?utm_source=share&utm_medium=web2x&context=3
